osf/management/commands/approve_pending_schema_responses.py

```diff
@@ -0,0 +1,63 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from django.utils import timezone
+from framework.celery_tasks import app as celery_app
+
+from osf.models import SchemaResponse
+from osf.utils.workflows import ApprovalStates
+
+from website.settings import REGISTRATION_APPROVAL_TIME
```

Comment:
Do we really want these to be the same? Is there a reason these are both 2 days?